src/styles/index.scss

```diff
@@ -1,3 +1,31 @@
+* {
+  box-sizing: border-box;
+}
+
+@import "./blocks/variables";
+
+@import "./blocks/bodyreset";
+
+@import "./variables";
+
 body {
   margin: 0;
 }
+
+@import "./blocks/cards";
+
+@import "./blocks/stars";
+
+@import "./blocks/resetting";
+
+button {
+  padding: 0;
+}
+
+h3 {
+  margin-bottom: 0;
+}
+
+p {
+  margin: 0;
+}
```

Comment:
you have the same code in `resetting.scss` (it's not a block actually, so move it out of the folder) and here - remove it here